policies/rudderc/docs/src/test.md

```diff
@@ -61,5 +61,5 @@ rudderc test --library /path/to/lib
 rudderc test --library /path/to/lib case1
 ```
 
-* the library parameter here **must** be a full library (containing `10_ncf_internals`, etc.), because it will also be used by the agent to run.
+* the library parameter here **must** be a path to a local full `ncf` library (containing `10_ncf_internals`, etc.), because it will also be used by the agent to run.
```

Comment:
This is unclear to me, do we have to pass the path to a ncf repo clone, a ncf repo tree or a ncf as deployed in the rudder-server package?

Reply:
any `tree` folder or equivalent (`/var/rudder/common`) is fine, I'll try to clarify this.